core/api/v1/keytransparency_proto/keytransparency.proto

```diff
@@ -273,6 +298,16 @@ service KeyTransparencyService {
     option (google.api.http) = { get: "/v1/domains/{domain_id}/epochs:stream" };
   }
 
+  // ListMutations returns a list of mutations in a specific epoch.
+  rpc ListMutations(ListMutationsRequest) returns (ListMutationsResponse) {
+    option (google.api.http) = { get: "/v1/domains/{domain_id}/epochs/{epoch}/mutations" };
+  }
+
+  // ListMutationsStream is a streaming list of mutatinos in a specific epoch.
```

Comment:
sp. 'mutatinos'  -  this is some newly discovered elementary particle right?

Reply:
Lol.  The elementary particles discovered when typing too late at night :-)

Reply:
Fixed.